DB/src/main/java/io/deephaven/db/v2/locations/parquet/topage/ToDbArrayPage.java

```diff
@@ -9,35 +9,37 @@
 import java.lang.reflect.Array;
 import java.nio.IntBuffer;
 
-public class ToDbArrayPage<ATTR extends Attributes.Any, RESULT> extends ToPage.Wrap<ATTR, RESULT, DbArrayBase[]> {
+public class ToDbArrayPage<ATTR extends Attributes.Any, RESULT, ARRAY_TYPE extends DbArrayBase<?>> extends ToPage.Wrap<ATTR, RESULT, ARRAY_TYPE[]> {
```

Comment:
If you take extends to a different line I think this is going to be easier to read.